src/styles.css

```diff
@@ -14,6 +14,15 @@
   --color-link-hover: var(--color-dim);
 }
 
+[data-theme="dark"] main blockquote {
+  font-size: 0.95rem;
+  color: #f1f1f1;
+}
+
+[data-theme="dark"] .footnote {
+  color: #f1f1f1;
+}
+
```

Comment:
Ok, I see that footnotes could be easier to read in dark mode. But these changes should be applied to the Lume's design system (in the `.footnotes` component: https://github.com/lumeland/ds/blob/main/src/components/footnotes.css).

And the same for blockquotes: https://github.com/lumeland/ds/blob/main/src/components/blockquote.css

And both colors and font sizes must be applied using [the existing CSS variables](https://github.com/lumeland/ds/blob/main/src/variables.css), so people can override them to customize the appearance.

Can you remove the changes in this file? So we can apply them in the lumeland/ds package.